doc/source/user_guide/changelog/v2.0.0.rst

```diff
@@ -0,0 +1,78 @@
+What's new in 2.0.0 (16 MAY 2025)
+-----------------------------------
+
+These are the changes in icepyx 2.0.0 See :ref:`release` for a full changelog
+including other versions of icepyx.
+
+.. warning::
+    This is major release and contains a few breaking changes. Full details will be
+    described in the documentation.
+
+    NSIDC (the NASA DAAC that hosts ICESat-2 data) is transitioning their
+    API endpoint from the on-premises EGI system to the cloud-based Harmony services.
+    icepyx has been updated to access these new services; Certain capabilities are temporarily going away
+    * Data reformatting: users will only get HDF5 files back from the new API.
```

Comment:
Need to add a blank line before the list, otherwise the list doesn't show up properly as bullet points.

```suggestion
    icepyx has been updated to access these new services; Certain capabilities are temporarily going away

    * Data reformatting: users will only get HDF5 files back from the new API.
```